pages/index.tsx

```diff
@@ -189,13 +197,22 @@ const Home: NextPage = () => {
               setNewPost(e.target.value)
             }}
           />
+          <textarea 
+            value={tag}
+            maxLength={100}
+            placeholder="Enter a Category Tag"
```

Comment:
This shouldn't keep us from merging this PR, but I wonder if we can get a more descriptive name to represent what category tag stands for? When I started up the app I wasn't really sure I understood what should I put there :)

Reply:
Related to this - the tag doesn't show up when I go to the "Explore page". See the attached file for an example:
<img width="926" alt="Screenshot 2023-09-13 at 17 23 48" src="https://github.com/ceramicstudio/EthDenver2023Demo/assets/7502466/f4901084-f173-4db5-90e9-9785442f74c7">

Reply:
Hi @JustinaPetr ! Thanks for your review - I sent up another few commits just now removing the config file and the admin seed. I also added conditional "tag" render logic (though I'm not sure how it's possible that the post would be rendering at all since the tag is a required field according to the schema).

Regarding the purposing of Tag - I'm not entirely sure what this is meant to represent. @velvet-shark  (I think) added this in during a more recent round of updates to the models (in order to give us more to filter/order against to display QFO capabilities)